torch/csrc/jit/passes/python_print.cpp

```diff
@@ -724,17 +802,17 @@ struct PythonPrintPass {
           assignValue(graph->inputs().at(i), node->inputs().at(i));
         }
         printBody(graph->block());
-        std::stringstream ss;
-        ss << "fork(" << name << ")";
-        printOutputDefinition(node, ss.str());
+        auto ss = std::make_shared<TaggedStringStream>(&source_range_stack_);
```

Comment:
Why does this need to be a TaggedStream? from its creation to its use, the actual source range is never changed.